### PR TITLE
add xcat in docker testcases

### PR DESCRIPTION
**Pull request by junxiawang**
@tingtli  please help review test case for docker in xcat

### CODE REVIEW

File: xCAT-test/autotest/testcase/installation/ubuntu_diskless_installation_docker [hunk 1]
```diff
@@ -0,0 +1,42 @@
+start:ubuntu_diskless_installation_docker
+os:Linux
+stop:yes
+
+cmd:copycds $$ISO
+check:rc==0
+cmd: chdef -t node -o $$DOCKERXCATCN vmstorage=dir:///var/lib/libvirt/images/ && rmvm $$DOCKERXCATCN -f -p &&  mkvm $$DOCKERXCATCN -s 15G
```
Comment (tingtli) [inline xCAT-test/autotest/testcase/installation/ubuntu_diskless_installation_docker]: @junxiawang  where did you define $$DOCKERXCATCN? Will you use $$CN as $$DOCKERXCATCN?

**Comment by zet809** [comment]
@junxiawang Hi, Jun xia, will you pls drive the reviewing of this pull request?

File: xCAT-test/autotest/testcase/installation/ubuntu_diskless_installation_docker (hunk 1 above)
Comment (junxiawang) [inline xCAT-test/autotest/testcase/installation/ubuntu_diskless_installation_docker]: We do not need to do any action for docker vm setup,for in docker cases using $$CN .

**Comment by whowutwut** [comment]
I don't think docker is priority, this pull request has been open for a long time. Are we planning to execute docker test cases in daily regression?  If not, perhaps we should close the pull request and not spend anymore time here.   If priority changes, we can re-look at this again.  If you don't agree, please re-open.

**Pull request closed without merging**